front/app/components/Areas/UpdateFollowArea.tsx

```diff
@@ -21,10 +22,14 @@ const UpdateFollowArea = ({ area }: Props) => {
   const { mutate: deleteFollower, isLoading: isDeletingFollower } =
     useDeleteFollower();
   const { pathname } = useLocation();
+  const theme = useTheme();
   const isLoading = isAddingFollower || isDeletingFollower;
   const followerId = area.relationships.user_follower?.data?.id;
   const isFollowing = !!followerId;
-  const color = isFollowing ? colors.success : colors.coolGrey300;
+  const areaButtonContentColor = isFollowing
+    ? colors.white
+    : theme.colors.tenantPrimary;
+  const className = isFollowing ? 'inverse' : '';
```

Comment:
When a variable is used only once, I normally don't make one not to clutter the component too much unless the variable name clarifies the code. Here, I would pass the value directly as a prop. But up to you.

Reply:
The same remark for the other className variables in this PR :)